packages/@icon-magic/generate/src/plugins/svg-to-raster.ts

```diff
@@ -104,19 +104,18 @@ async function generatePng(
   height: number,
   outputPath: string
 ): Promise<void> {
-  const png = await convert(svg, { width, height });
-  await fs.writeFile(outputPath, png);
+  await convertToPng(svg, { width, height }, outputPath);
 }
 
+
 /**
- * Converts a png file to webp and writes to the outputPath
- * @param input path to the input file in .png format
- * @param outputPath path where the output needs to go
+ * Generate a webp asset from an svg string
+ * @param svg the contents of an svg file that needs to be converted to png
+ * @param outputPath path to write the webp file after it's created
  */
-function convertToWebp(pathToPng: string, outputPath: string): Promise<string> {
-  return new Promise<string>((resolve, reject) => {
-    webp.cwebp(pathToPng, outputPath, '-q 80', function(status: string) {
-      !!~status.indexOf('100') ? resolve(outputPath) : reject();
-    });
-  });
-}
+async function generateWebp(
```

Comment:
We don't need to create a wrapper around this. We can call it directly on line 65.